credentials/utopia-birth-certificate-vc-v2/queries.json

```diff
@@ -0,0 +1,24 @@
+{
+  "default": {
+    "type": "QueryByExample",
+    "credentialQuery": [
+      {
+        "reason": "Please present your Utopia Birth Certificate Verifiable Credential(s) to complete the verification process.",
+        "example": {
+          "@context": [
+            "https://www.w3.org/ns/credentials/v2",
+            "https://examples.vcplayground.org/contexts/shim-render-method-term/v1.json",
+            "https://w3id.org/vc/render-method/v2rc1"
+          ],
+          "type": ["UtopiaBirthCertificateCredential"]
```

Comment:
Is this treated as JSON or JSON-LD?  Does it need to be defined somewhere too as in the credential?